<?xml version='1.0' encoding='UTF-8' standalone='no'?>
<doxygen xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="compound.xsd" version="1.9.4" xml:lang="en-US">
  <compounddef id="classGambit_1_1ColliderBit_1_1Py8Collider_1_1EventGenerationError" kind="class" language="C++" prot="public">
    <compoundname>Gambit::ColliderBit::Py8Collider::EventGenerationError</compoundname>
    <basecompoundref prot="public" virt="non-virtual">std::exception</basecompoundref>
    <includes refid="Py8Collider_8hpp" local="no">Py8Collider.hpp</includes>
      <sectiondef kind="private-func">
      <memberdef kind="function" id="classGambit_1_1ColliderBit_1_1Py8Collider_1_1EventGenerationError_1ab5e03b071d35c4442b1693db4d5fca16" prot="private" static="no" const="yes" explicit="no" inline="yes" virt="virtual">
        <type>const char *</type>
        <definition>virtual const char * Gambit::ColliderBit::Py8Collider&lt; PythiaT, EventT, hepmc_writerT &gt;::EventGenerationError::what</definition>
        <argsstring>() const</argsstring>
        <name>what</name>
        <qualifiedname>Gambit::ColliderBit::Py8Collider::EventGenerationError::what</qualifiedname>
        <exceptions> throw ()</exceptions>
        <briefdescription>
        </briefdescription>
        <detaileddescription>
        </detaileddescription>
        <inbodydescription>
        </inbodydescription>
        <location file="C:/Hugo/Sites/gambit_2.2_sphinx/ColliderBit/include/gambit/ColliderBit/colliders/Pythia8/Py8Collider.hpp" line="73" column="30" bodyfile="C:/Hugo/Sites/gambit_2.2_sphinx/ColliderBit/include/gambit/ColliderBit/colliders/Pythia8/Py8Collider.hpp" bodystart="73" bodyend="76"/>
      </memberdef>
      </sectiondef>
    <briefdescription>
<para>An exception for when Pythia fails to generate events. </para>
    </briefdescription>
    <detaileddescription>
    </detaileddescription>
    <inheritancegraph>
      <node id="1">
        <label>Gambit::ColliderBit::Py8Collider&lt; PythiaT, EventT, hepmc_writerT &gt;::EventGenerationError</label>
        <link refid="classGambit_1_1ColliderBit_1_1Py8Collider_1_1EventGenerationError"/>
        <childnode refid="2" relation="public-inheritance">
        </childnode>
      </node>
      <node id="2">
        <label>std::exception</label>
      </node>
    </inheritancegraph>
    <collaborationgraph>
      <node id="1">
        <label>Gambit::ColliderBit::Py8Collider&lt; PythiaT, EventT, hepmc_writerT &gt;::EventGenerationError</label>
        <link refid="classGambit_1_1ColliderBit_1_1Py8Collider_1_1EventGenerationError"/>
        <childnode refid="2" relation="public-inheritance">
        </childnode>
      </node>
      <node id="2">
        <label>std::exception</label>
      </node>
    </collaborationgraph>
    <location file="C:/Hugo/Sites/gambit_2.2_sphinx/ColliderBit/include/gambit/ColliderBit/colliders/Pythia8/Py8Collider.hpp" line="71" column="9" bodyfile="C:/Hugo/Sites/gambit_2.2_sphinx/ColliderBit/include/gambit/ColliderBit/colliders/Pythia8/Py8Collider.hpp" bodystart="72" bodyend="77"/>
    <listofallmembers>
      <member refid="classGambit_1_1ColliderBit_1_1Py8Collider_1_1EventGenerationError_1ab5e03b071d35c4442b1693db4d5fca16" prot="private" virt="virtual"><scope>Gambit::ColliderBit::Py8Collider::EventGenerationError</scope><name>what</name></member>
    </listofallmembers>
  </compounddef>
</doxygen>
